Lib/test/support/__init__.py

```diff
@@ -58,7 +58,8 @@
     "LOOPBACK_TIMEOUT", "INTERNET_TIMEOUT", "SHORT_TIMEOUT", "LONG_TIMEOUT",
     "Py_DEBUG", "EXCEEDS_RECURSION_LIMIT", "Py_C_RECURSION_LIMIT",
     "skip_on_s390x",
-    "without_optimizer",
+    "without_optimizer", "reset_colorized_globally",
```

Comment:
Also not used anywhere and fails `test___all__`:
```suggestion
    "without_optimizer",
```